### PR TITLE
Adds both a verb and action button for drones to be able to view station alarms.

### LINKED ISSUE #11598: Drones do not have a check alerts button
Drones get messages if station alarms go off, but unlike borgs and Ais, they do not have a button to check what alerts are currently active, nor do they have a verb to check it. This is an issue because if a drone didn't see the alarm first go off, they have no idea that there is an issue in that area.

### PR DESCRIPTION
**Pull request by Firecage**
Previously Drones, while receiving alerts about power, air, and something else I forgot, they were not actually able to view these alerts. This corrects that. It gives them both a verb and an action button to view their own limited station alerts.

Fixes #11598

:cl: Firecage
add: We at NanoTrasen R&D would like to announce a new software upgrade for all drones! They now have the capability to view station alerts!
/:cl:

### CODE REVIEW

File: code/modules/mob/living/simple_animal/friendly/drone/_drone.dm [hunk 1]
```diff
@@ -91,6 +91,8 @@
 		SF.Grant(src)
 	else
 		verbs -= /mob/living/simple_animal/drone/verb/toggle_statics
+	var/datum/action/generic/drone/show_alerts/SA = new(src)
```
Comment (ChangelingRain) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: cogscarabs shouldn't get this or the verb

Comment (Firecage) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: Ah, right, I forgot they were children of drones. I'le fix that tomorrow.

**Comment by optimumtact** [comment]
fuk drones

File: code/modules/mob/living/simple_animal/friendly/drone/_drone.dm [hunk 2]
```diff
@@ -64,6 +64,7 @@
 	var/obj/item/default_storage = /obj/item/weapon/storage/backpack/dufflebag/drone //If this exists, it will spawn in internal storage
 	var/obj/item/default_hatmask //If this exists, it will spawn in the hat/mask slot if it can fit
 	var/seeStatic = 1 //Whether we see static instead of mobs
+	var/seeAlerts = 1
```
Comment (RemieRichards) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: ```TRUE```/```FALSE``` please, same for ```seeStatic```, if you would please.

Comment (Firecage) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: @RemieRichards Honest question. What exactly is the difference between using TRUE / FALSE and 1 / 0?

Comment (optimumtact) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: @Firecage nothing, in the end it compiles down to 1 and 0

The important thing is that if we see TRUE/FALSE we can more easily make the assumption that the variable is a boolean on/off flag of some description

It also makes the code nicer to read.

Comment (Firecage) [inline code/modules/mob/living/simple_animal/friendly/drone/_drone.dm]: @optimumtact Atleast I don't have to go change all those on/off's in the code to TRUE/FALSE from 1/0!

**Comment by Shadowlight213** [comment]
:+1: I recall trying to do this a while ago using the same system silicons had and failing horribly.

**Comment by Firecage** [comment]
@Shadowlight213 The AI or the Cyborg one? I used the system which Cyborgs had for this.

**Comment by AnturK** [comment]
Please don't use action buttons for static interface elements. If something should be always visible for the given mobtype just add dedicated ui element.

**Comment by coiax** [comment]
Indeed, it should be a dedicated UI element. Even though it's much harder to make those than action buttons.

**Comment by RemieRichards** [comment]
@coiax not really, it's no harder.

**Comment by Firecage** [comment]
@AnturK I have no idea how to do any hud or UI stuff at all.

**Comment by Shadowlight213** [comment]
@Firecage the cyborg one.
I was having issues getting it to actually update with data.

**Comment by optimumtact** [comment]
Enough maintainers said they wanted a built in UI item instead so closing this.

**Pull request closed without merging**